docs/reference/tab-widgets/quick-start-install.asciidoc

```diff
@@ -0,0 +1,48 @@
+
+// tag::cloud[]
+include::{docs-root}/shared/cloud/ess-getting-started.asciidoc[tag=generic]
+// end::cloud[]
+
+// tag::self-managed[]
+**Install and run {es} using Docker**
```

Comment:
There are also instructions for using Docker compose to spin up multiple ES instances and Kibana so you have the equivalent of a cloud deployment. https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-docker.html

Reply:
Agree that it makes things more complex--mostly mentioned in case you wanted to cross reference it somehow. (Though that's probably not a great idea, as you don't want to send folks off somewhere else.)